handbook/Best practices/TypeScript/README.md

```diff
@@ -0,0 +1,30 @@
+# TypeScript
+
+[TypeScript](https://www.typescriptlang.org/) is a typed superset of JavaScript that we strongly advise projects to use in favour of vanilla JavaScript. This enables projects to leverage TypeScript's compiler to help catch a number of potential bugs and errors in projects well before it ships. In complement with TypeScript, the [VSCode editor](https://code.visualstudio.com/docs/languages/typescript) provides TypeScript language-specific features such as inline feedback from TypeScript typechecker. 
```

Comment:
👏 

Do we have a decision record we can link to for TS?

Reply:
We don't and I'm quite surprised we don't